src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir-normal;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.bigField {
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: Avenir-normal, serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.bigField:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.bigField:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-family: Avenir-bold, serif;
+  font-style: normal;
+}
+
+.smallField {
+  width: 100%;
+  height: 42px;
+  font-family: Avenir-normal, serif;
```

Comment:
There are a some similar styles for `bigField` and `smallField`, let's improve this to not repeat these styles, and use the approach from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

![Screenshot 2023-07-10 at 20 57 02](https://github.com/mate-academy/layout_search-bar-airbnb/assets/32853034/19310770-89ad-4c2e-a976-9e484de931d3)
![Screenshot 2023-07-10 at 20 58 02](https://github.com/mate-academy/layout_search-bar-airbnb/assets/32853034/5eb94068-6e7e-4f7e-abfb-b53a418fb149)